active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,575 @@
+
+### Add a margin 
+
+A teaching tip can have a margin added by setting the Margin property. When a teaching tip is attached to a UI element, a margin will move it away from the UI element it is pointing to. When a teaching tip is not attached to anything, a margin will move it away from the side of the app window it is positioned towards. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick"></Button>
+<muxc:TeachingTip> 
+    <muxc:TeachingTip x:Name="AutoSaveTip"
+        Title="Saving automatically"
+        Subtitle="We save your changes as you go - so you never have to."
+        Margin="80">
+    </muxc:TeachingTip>
+</muxc:TeachingTip.Attach>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipMarginSample.jpg" height="600"></a>
+
+### Point to a UI element
+
+You can change the teaching tip to point to a specific UI element by setting the TeachingTipPlacementMode property. For a targeted teaching tip, this placement will be relative to the UI element target. For a non-targeted teaching tip, this placement will relative to app window.  
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipPointerSample.jpg" height="600"></a>
+
+### Add content
+
+Content can be added to a teaching tip using the Content property. If there is more content to show than what the size of a teaching tip will allow, a scrollbar will be automatically enabled to allow a user to scroll the content area. 
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+                <StackPanel>
+                    <CheckBox Content="Don't show tips at start up" Checked="HandleChecked" Unchecked="HandleUnchecked" />
+                    <TextBlock>You can change your tip preferences in <Hyperlink NavigateUri="app:/item/SettingsPage">Settings</Hyperlink> if you change your mind.</TextBlock>
+                </StackPanel>
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
```

Comment:
Show potential code (even if just comments) for the checkbox too or instead how to catch value of checkbox when the teaching tip is dismissed would be much more useful than having to track state as the checkbox is interacted with.

Reply:
Thanks for this feedback, @michael-hawker! I will be passing over all of these examples again now that I have more details pinned down.

Reply:
Thanks again, @michael-hawker, I ran this by our representatives for guidelines and documentation and added what they felt was the appropriate level of detail to include - which was setting and using an x:bound Boolean with the CheckBox to toggle whether tips show. Our samples in the XAML sample gallery will be more complete.